### ISSUE TITLE
Uptime robot

### ISSUE THREAD

**Issue by averyanalex** [yggdrasil-network/public-peers#181]
How an idea to do uptime robot on python? Someone, say me, how to understand, node online or not?
[https://publicpeers.neilalexander.dev/publicnodes.json](https://publicpeers.neilalexander.dev/publicnodes.json) maybe this?
If you lke this idea, I will do so.

**Comment by Saiv46** [comment]
Ahem, there's already a page for that: https://publicpeers.neilalexander.dev/

**Comment by averyanalex** [comment]
Yes, I know, but there is only online and offline status. No uptime in percents
